### PR TITLE
fix history panel not rendering when history is empty

### PR DESCRIPTION
**Pull request by basokant**
Provides fix for #1088 

Freshly initialized repository after changes:

1088-fix

### CODE REVIEW

File: src/model.ts [hunk 1]
```diff
@@ -919,14 +919,18 @@ export class GitExtension implements IGitExtension {
...
+          return await requestAPI<Git.ILogResult>(
+            URLExt.join(path, 'log'),
+            'POST',
+            {
+              history_count: count,
+              follow_path: this.selectedHistoryFile?.to
+            }
+          );
+        } catch (error) {
+          return { code: 0, commits: [] };
```
Comment (fcollonval) [inline src/model.ts:932]: As the code in the panel is:

https://github.com/jupyterlab/jupyterlab-git/blob/c7e7363a5e749ea4590984f3a0303694f25212aa/src/components/GitPanel.tsx#L264-L270

The best is to return a non-null code (`1` e.g) and the state will be an empty array as wanted.

Comment (basokant) [reply src/model.ts]: Pushed this change! Thanks for the guidance @fcollonval!

**Review by fcollonval — APPROVED** [review]
Thanks a lot

Could you confirm me you work with @shawnesquivel? If yes, I will add him as co-author.

**Comment by basokant** [comment]
> Thanks a lot
> 
> Could you confirm me you work with @shawnesquivel? If yes, I will add him as co-author.

Yes @shawnesquivel and I both worked on the initial fix, but he wasn't able to submit the PR due to an issue with running the pre-commit hook, so I ended up submitting it.

**Comment by fcollonval** [comment]
@all-contributors please add @basokant for code

**Comment by fcollonval** [comment]
https://github.com/all-contributors please add @shawnesquivel for code

**Comment by fcollonval** [comment]
@all-contributors please add @shawnesquivel for code

**Pull request merged**